### ISSUE TITLE
The java script links don't have a unique and meaningful description in Progress tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1945]
URL: http://clarity-ks.surge.sh/progress-bars

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Progress tab.
3. Check the java script links in Progress tab.

Actual Results:
The two java script links have same description “link”, and the description is not meaningful.  
The other java script link “Another link” does not have meaningful description.
For more details, please refer to “Progress-JavascriptLinks-DonotHaveUniqueAndMeaningfulDescription.png”.

Expected Results:
All java script links should have unique and meaningful description.  

Violation of Section 508 Standards: 
Section 1194.22(I) Frames shall be titled with text that facilitates frame identification and navigation. 
Section 1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
Section 1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind.

Screen shot:
1.	Progress-JavascriptLinks-DonotHaveUniqueAndMeaningfulDescription.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/f11b1204-16b2-4522-a2f4-8c1342b80311)
